packages/app/src/specs/banners/RecordBanner.vue

```diff
@@ -58,29 +58,44 @@ query RecordBanner {
 }
 `
 
-withDefaults(defineProps<{
+const props = withDefaults(defineProps<{
   modelValue: boolean
   hasBannerBeenShown: boolean
+  commandOptions?: CohortOption[]
 }>(), {
   modelValue: false,
   hasBannerBeenShown: true,
+  commandOptions: () => {
+    return [
+      { cohort: 'A', value: 'cypress' },
```

Comment:
Is running `cypress` without `npx` really advisable? Most people won't have it installed globally. Also, I did not see this A/B test specified in the docs of Figma - is it there somewhere?

Reply:
The requirements for the A/B testing of the command was defined in the document describing the other A/B copy options here: https://docs.google.com/document/d/1PFN79d3deDyqDVFmVIBe4cVMzWiEll0J6XCpesBZrSg/edit#

The documentation here (https://docs.cypress.io/guides/guides/command-line) mentions `npx` along side the various other ways to run cypress.  Currently, the `<RunsEmpty />` component just shows the command without the `npx` option on it. 

Do you think we should be encouraging one use over another?

Reply:
@warrensplayer there's no way to track the performance of the A/B test of this record run step, right? If so, I think we should just not do it and show `npx` for all cases.

Reply:
@pstakoun That is correct.  We cannot track the performance of users running these command line options at this time.  I will remove this for now.

Reply:
@pstakoun On second thought, I will discuss with you shortly during our meeting on options.